src/handlers/metagraphHandler.ts

```diff
@@ -363,9 +363,9 @@ export const currencyTransactionsByDestination = async (
   }
 };
 
-const balanceOrZeroFn = async (balance, address) => {
+const balanceOrZeroFn = async (balance, address, ordinal) => {
   if (balance === null) {
-    const snapshot_ordinal = (await latestMetagraphSnapshot())?.ordinal;
+    const snapshot_ordinal = isFinite(ordinal)? ordinal:(await latestMetagraphSnapshot())?.ordinal;
     return { balance: 0, address, snapshot_ordinal };
```

Comment:
`snapshot_ordinal` needs to be `ordinal` to match the current API spec.

Reply:
This is slightly annoying, I know. I use the same name as the db column, so if the balance is found in the database, I don't need to touch it.
At the end, it stills gets transformed to ordinal in the response producer:
```
export const balanceResponse = (balance) => ({
  ordinal: balance.snapshot_ordinal,
  balance: balance.balance,
  address: balance.address,
});
```
The final output is ordinal anyway.